rust/src/conf.rs

```diff
@@ -142,6 +143,22 @@ impl ConfNode {
         return false;
     }
 
+    // Get a child node of this node by name.
+    //
+    // Wrapper around ConfNodeLookupChild.
+    //
+    // Returns None if the child is not found.
+    pub fn get_child(&self, name: &str) -> Option<ConfNode> {
+        unsafe {
+            let name = CString::new(name).unwrap();
+            let child = ConfNodeLookupChild(self.conf, name.as_ptr());
+            if child.is_null() {
+                None
+            } else {
+                Some(ConfNode { conf: child })
+            }
+        }
+    }
```

Comment:
Not used by our Rust, but still an existing conf function that was never exposed to Rust. But useful to have.